modules/@apostrophecms/page/index.js

```diff
@@ -597,6 +643,15 @@ database.`);
       async patch(req, _id) {
         return self.withLock(req, async () => {
           const input = req.body;
+          const keys = Object.keys(input);
+          let possiblePatchedFields;
```

Comment:
Avoids updating updatedAt just because of a lock. Already had this fix in pieces.